src/Application.vala

```diff
@@ -87,6 +89,9 @@ public class ENotes.Application : Granite.Application {
     public override void activate () {
         if (!running) {
             settings = ENotes.Services.Settings.get_instance ();
+            
+            clockSettings = new ClockSettings ();
+            use24HSFormat = (clockSettings.clock_format == "24h");
```

Comment:
Since we will be now "depending" on a settings schema that we don't include, you need to first validate that it exists, if not it will fail :) 

See this code for how it's done: https://github.com/elementary/wingpanel-indicator-nightlight/blob/master/src/Indicator.vala#L106